### PR TITLE
Code Review Changes

### PR DESCRIPTION
**Pull request by Sambaba23**
Analysis of the program.
Overall this program contained was coded really well and I was able to understand it well. I had to really look hard to get suggestions in. 
Was the program available in UC Github on time? Yes, it was available on time.
Is the program easy for you, as an outsider, to understand? Yes the program was easy to understand.
Does the program compile? Yes this program can compile.
Links to three commits you made to your group's repository during Sprint 1.
https://github.com/brennerdanae/CachingInApp/commit/f11d942e94cff2e218e49a84a414895872c6c8ed
https://github.com/brennerdanae/CachingInApp/commit/dc506988679f7ee68472126d1bfe150ee80b9e3d
I forgot to press the commit and pus button for this commit.

Technical concepts I learned
I learned how tests work in Java 
```
@Test
    public void shouldReturnDefaultMessage() throws Exception {
        this.mockMvc.perform(get("/")).andDo(print()).andExpect(status().isOk())
                .andExpect(content().string(containsString("Hello World")));
    }

```
Setting variables for DTO interface

```
@NonNull private String id;
    @NonNull private String groupName;
    @NonNull private String companyName;
    @NonNull private String location;
    @NonNull private Date startDate;
    private String groupContact;
    private String contactInformation;
    private Boolean authorized;
    private Integer numberOfWorkers;
    private Map<String, String> suppliesNeeded;
```
linking bootstrap in html
` <link th:href="@{/css/bootstrap.min.css}" rel="stylesheet" />`

### CODE REVIEW

File: src/main/java/com/laborhelper/enterprise/LaborHelperController.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
+
```
Comment (Sambaba23) [inline src/main/java/com/laborhelper/enterprise/LaborHelperController.java:5]: Removed this import as it wasn't in use.
https://martinfowler.com/bliki/TechnicalDebt.html

File: src/main/resources/templates/layout.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <title>Title</title>
     <link th:href="@{/css/main.css}" rel="stylesheet" />
-
+    <link th:href="@{/css/bootstrap.min.css}" rel="stylesheet" />
```
Comment (Sambaba23) [inline src/main/resources/templates/layout.html:7]: Add a link to the bootstrap style sheet.

File: src/main/resources/templates/layout.html [hunk 3]
```diff
@@ -18,9 +18,9 @@
         </button>
         <div class="collapse navbar-collapse" id="navbarCollapse">
 
-            <form id="zipsearch"  class="d-flex align-content-center" role="search">
+            <form id="zipSearch"  class="d-flex align-content-center" role="search">
```
Comment (Sambaba23) [inline src/main/resources/templates/layout.html:21]: Added in camel case for the variables

File: src/main/resources/templates/registration.html [hunk 4]
```diff
@@ -13,10 +14,12 @@
 
     <ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
       <li class="nav-item text-center">
-        <a class="nav-link active btl" id="pills-home-tab" data-toggle="pill" href="#pills-home" role="tab" aria-controls="pills-home" aria-selected="true">Login</a>
+        <a class="nav-link active btl" id="pills-home-tab" data-toggle="pill" href="#pills-home" role="tab"
```
Comment (Sambaba23) [inline src/main/resources/templates/registration.html:17]: added in proper indentation.

https://www.baeldung.com/java-clean-code

File: src/main/java/com/laborhelper/enterprise/LaborHelperController.java (hunk 1 above)
Comment (discospiff) [reply src/main/java/com/laborhelper/enterprise/LaborHelperController.java:5]: Use Control-Shift-O to have the IDE organize imports for you.

File: src/main/resources/templates/registration.html [hunk 5]
```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
-<html lang="en" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
-      layout:decorate="~{layout}">
+<html lang="en" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout">
```
Comment (discospiff) [inline src/main/resources/templates/registration.html:2]: Why remove layout:decorate?

**Comment by discospiff** [comment]
I recommend that the group review each of these changes and decide what to use.  See comments in Files Changed.

**Pull request closed without merging**